README.md

```diff
@@ -211,14 +218,52 @@ We will create a method called `startMonitoring` to start the monitoring process
 
 ```swift
   func startMonitoring() {
-    trusona.monitorPendingTrusonafications(
-      onCompleted: { result in
-        self.dismiss(animated: true)
-      },
-      failure: { error in
-        self.show(message: "Failed to get pending trusonafications: \(error)")
+        trusona.monitorPendingTrusonafications(onCompleted: { (result) in
+        
+        switch result {
+        case .success:
+          // Trusonafication was successfully accepted and validated.
+          print("Accepted")
+          
+        case .error(let type):
+          // Trusonafication failed to process in the back end, handle the error here
+          
+          switch type {
+          case .accepted:
+            // Trusonafication was already accepted
+            print("Accepted")
+            
+          case .canceled:
+            // Trusonafication was already cancelled
+            print("Cancelled")
+            
+          case .invalid:
+            // Trusonafication was invalid
+            print("Invalid")
+            
+          case .rejected:
+            // Trusonafication was rejected
```

Comment:
Maybe use `// Trusonafication was already rejected` so this case will not be confused with the one on line 257?

Reply:
ok, fixing it

Reply:
fixed